### PR TITLE
Override layout for Key Metrics widgets at mobile viewports to be full width.

### PR DESCRIPTION
**Pull request by techanvil**
## Summary

Addresses issue:

- #7159 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @techanvil ! This is very close however, on further inspection I noticed that the design shows that in mobile the title and content have an even split of the row while remaining left-aligned. Currently the two are justified to the edges.

This is easier to see on a wide mobile viewport around 550px or so.

After some tinkering, I found that the following additional style seems to do the trick but please give it a second look.

```css
.googlesitekit-km-widget-tile__title,
.googlesitekit-km-widget-tile__body {
    flex: 1;
}
```

**Comment by techanvil** [comment]
**Update:** So, further to the above, I have in fact noticed that in fact an additional change _is_ needed, and I've applied a [fix](https://github.com/google/site-kit-wp/pull/7261/commits/87d91d1d848828c0dff7c2d9c75c90e003b43b88), to add some padding between the title and body in the mobile view (for numeric/text tiles), in order to avoid this scenario where there's no space between them:

image

**Comment by aaemnnosttv** [comment]
> So, further to the above, I have in fact noticed that in fact an additional change _is_ needed, and I've applied a [fix](https://github.com/google/site-kit-wp/pull/7261/commits/87d91d1d848828c0dff7c2d9c75c90e003b43b88), to add some padding between the title and body in the mobile view (for numeric/text tiles), in order to avoid this scenario where there's no space between them

Nice catch 👍 Just one last consideration.

File: assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss [hunk 1]
```diff
@@ -79,6 +86,17 @@
...
+	.googlesitekit-km-widget-tile--numeric,
+	.googlesitekit-km-widget-tile--text {
+		.googlesitekit-km-widget-tile__title {
+			padding-right: 12px;
+
+			@media (min-width: $bp-tablet) {
+				padding-right: 0;
+			}
+		}
+	}
```
[the remark this answers is not in the document]
Comment (techanvil) [reply assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss:98]: Hey @aaemnnosttv, `gap` was my first thought too as it's a cleaner solution on the CSS side of things, but I realised it results in the right-hand pane being aligned slightly off-center. Using padding instead is a better match to Figma where the right-hand content starts in the center of the widgets...

[the remark this answers is not in the document]
Comment (aaemnnosttv) [reply assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss:98]: SGTM 👍

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks @techanvil !

**Pull request merged**